Examples/restcomm-olympus/app/build.gradle

```diff
@@ -67,6 +67,7 @@ android {
             // Not specifying applicationIdSuffix since release uses the default applicationId: org.restcomm.android.olympus
             buildConfigField "boolean", "ENABLE_TEST_FAIRY_RUNTIME", "true"
             buildConfigField "String", "TESTFAIRY_APP_TOKEN", TESTFAIRY_APP_TOKEN
+            buildConfigField "String", "TEST_PUSH_FCM_KEY", TEST_PUSH_FCM_KEY
```

Comment:
Now I see why you wanted to take the TEST_ out :). But since we 'll be going over productization soon we probably will separate test and production values. So let's leave it as is for now

Reply:
ok:)